app/models/prosecution_case.rb

```diff
@@ -4,7 +4,7 @@ class ProsecutionCase < ApplicationRecord
   validates :body, presence: true
 
   def defendants
-    body['defendants'].map { |defendant| Defendant.new(body: defendant) }
+    body['defendants']&.map { |defendant| Defendant.new(body: defendant) }
```

Comment:
I'm wondering if we need a safe navigation operator here, considering that the [defendants are available in the schema](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/6955ae92a00efe5adc924eba2f5aaf3e658b97f9/lib/schemas/global/search/prosecutionCaseSummary.json#L39)

Reply:
Removed